### PR TITLE
Jay Shop Screen Complete Need Reviews

### PR DESCRIPTION
**Pull request by ttran0012**
Reviews for Jayshop screen

### CODE REVIEW

**Comment by ttran0012** [comment]
Did some code clean up and add a few things.

**Comment by ttran0012** [comment]
If you want to fix or make changes. Feel free! Cause Me is running out of time!

**Comment by Falzaro** [comment]
> If you want to fix or make changes. Feel free! Cause Me is running out of time!

Okay, I'll take a look at it today.

File: screens/Jayshop.jsx [hunk 1]
```diff
@@ -1,7 +1,22 @@
+<<<<<<< HEAD
+import React from 'react';
+import { StyleSheet, View, FlatList, Linking, ImageBackground , } from "react-native";
+=======
 import { StyleSheet, View } from "react-native";
 
+>>>>>>> main
```
Comment (Falzaro) [inline screens/Jayshop.jsx:7]: I'm still seeing merge conflicts. Please check the code that has <<< HEAD and resolve the conflicts.

Comment (ttran0012) [reply screens/Jayshop.jsx]: should be good to go now.

**Pull request merged**